roles-in-tech/developer-advocate.md

```diff
@@ -2,89 +2,173 @@
 
 ### Projected Time
 
-Example: 30-45 minutes
-- Lesson: ? min
-- Guided Practice: ?-? min
-- Independent Practice: ? min
-- Check for Understanding: ?-? min
+- Lesson: 10 min
+- Guided Practice: 30 min
+- Independent Practice: 10 min
 
-### Prerequisites
 
-Here are links to lessons that should be completed before this lesson:
-
-- Example: [Git version control](version-control/git-version-control/git-version-control.md)
 
 ### Motivation
 
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
+A developer evangelist is a spokesperson, mediator and translator between a company and both its technical staff and outside developers. Inorder to be a developer advocate one must have the following skills:
+irst there are the necessary skills:
```

Comment:
There is an error in line 14 `irst there are the necessary skills:` Can you update that content ?